src/Apps/Conversation/Components/DetailsHeader.tsx

```diff
@@ -21,13 +23,20 @@ interface DetailsHeaderProps extends DetailsProps {}
 
 export const DetailsHeader: FC<DetailsHeaderProps> = props => {
   const { showDetails, setShowDetails } = props
+  const isMobile =
+    __internal__useMatchMedia(themeProps.mediaQueries.xs) ||
+    __internal__useMatchMedia(themeProps.mediaQueries.sm)
```

Comment:
Both of these use `window` internally which might blow up SSR rendering; can you give that a test? 

Fine with using that here tho `__internal__` was meant to mean: don't use, its only for other internal implementations! But i can see it being useful occasionally (if its not breaking SSR rendering)